### PR TITLE
[docs] Fix article text in «The deckhouse-web module: usage» page

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

The "The deckhouse-web module: usage" article reviewing and fixing.

## Changelog entries

```changes
module: docs
type: fix
description: "Review and fix the 'The deckhouse-web module: usage' article."
```

### CODE REVIEW

File: modules/810-deckhouse-web/docs/USAGE.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: "The deckhouse-web module: usage"
 
 ## An example of the module configuration
 
+Below is a simple example of the module configuration:
```
Comment (z9r5) [inline modules/810-deckhouse-web/docs/USAGE.md:7]: @Zhbert no need to repeat the same text I think. The header contains the same text.

File: modules/810-deckhouse-web/docs/USAGE.md [hunk 2]
```diff
@@ -17,3 +19,5 @@ deckhouseWeb: |
     authSignInURL: "https://<applicationDomain>/sign-in"
     authResponseHeaders: "Authorization"
 ```
+
+You can also configure additional parameters. More information about this [in the next chapter](./configuration.html).
```
Comment (z9r5) [inline modules/810-deckhouse-web/docs/USAGE.md:23]: @Zhbert I think it is unnecessary to link to the configuration page here. It is not a guide or something like that.

File: modules/810-deckhouse-web/docs/USAGE.md (hunk 1 above)
Comment (Zhbert) [reply modules/810-deckhouse-web/docs/USAGE.md:7]: @z9r5 Maybe you're right. But I've always thought that a headline is a headline. After it, you need some kind of introduction. And the black square just behind the title looks ugly :)

File: modules/810-deckhouse-web/docs/USAGE.md (hunk 2 above)
Comment (Zhbert) [reply modules/810-deckhouse-web/docs/USAGE.md:23]: @z9r5 OK.

**Pull request merged**